webpack.config.js

```diff
@@ -2,8 +2,9 @@ const path = require('path')
 const webpack = require('webpack')
 const TerserPlugin = require('terser-webpack-plugin')
 
-// Set this to false for easier debugging:
-const production = true
+// DO NOT EDIT THIS CONSTANT
```

Comment:
Let's tell the use what *to* do, instead of what *not* to do. Maybe just replace these two lines with a simple `// Use "yarn prepare.dev" to make a debug-friendly build:`